share/extender/build_input.yml

```diff
@@ -122,15 +122,15 @@ platforms:
 
   osx:
     env:
-        PATH:                   "{{env.PLATFORMSDK_DIR}}/XcodeDefault{{env.XCODE_15_VERSION}}.xctoolchain/usr/bin:{{env.PATH}}"
+        PATH:                   "{{env.PLATFORMSDK_DIR}}/XcodeDefault{{env.XCODE_16_VERSION}}.xctoolchain/usr/bin:{{env.PATH}}"
         PLATFORMSDK_DIR:        "{{env.PLATFORMSDK_DIR}}"
         MANIFEST_MERGE_TOOL:    "{{env.MANIFEST_MERGE_TOOL}}"
-        XCODE_VERSION:          "{{env.XCODE_15_VERSION}}"
-        XCODE_CLANG_VERSION:    "{{env.XCODE_15_CLANG_VERSION}}"
-        MACOS_VERSION:          "{{env.MACOS_14_VERSION}}"
+        XCODE_VERSION:          "{{env.XCODE_16_VERSION}}"
+        XCODE_CLANG_VERSION:    "{{env.XCODE_16_CLANG_VERSION}}"
+        MACOS_VERSION:          "{{env.MACOS_15_VERSION}}"
         MACOS_VERSION_MIN:      "{{env.MACOS_VERSION_MIN}}"
         SWIFT_VERSION:          "{{env.SWIFT_5_5_VERSION}}"
```

Comment:
there is no `swift-6.0` folder (see line 137), not sure if swift version should be changed